src/lib/eliom_client.client.ml

```diff
@@ -402,15 +402,9 @@ let call_ocaml_service
       ?keep_nl_params ?nl_params ?keep_get_na_params
       ?progress ?upload_progress ?override_mime_type
       get_params post_params in
-  let%lwt () = Lwt_mutex.lock load_mutex in
```

Comment:
The ocaml service itself is not a page, but the lock could apply in scenarios that involve nesting of service calls. It sounds like we would somehow end up calling an OCaml service in the load phase of another page, and trouble would ensue?